### ISSUE TITLE
video publishing problems

### ISSUE THREAD

**Issue by eokeeffe** [AutonomyLab/ardrone_autonomy#84]
I was wondering if their needs to be any proper setup for the video publishing . I'm subscribing to ardrone/front and I find that I have to run the application sometimes 2-5 times to get the video to be published. Is this a common problem at the moment or do you think this might be more for my setup . I've checked my code and checked the publishing using the image_view package and the feed seems to have nothing on it. I find that if I restart my OS then it fixes  the problem generally. 

System : Ubuntu 12.04 LTS , ROS Groovy

**Comment by eokeeffe** [comment]
I've also noticed a small pattern , when the drone ROS driver is starting if the video_recorder thread ends itself , disconnecting and reconnecting the wifi connection while the driver is on seems to reinitialize the video stream. This is has been tested on drone firmware 2.2.9 and doesn't seem to work but works for the new 2.4.8 drone firmware

**Comment by oars** [comment]
I'm getting very slow frame rate, but it never failed to publish a video. Maybe 1 frame every 2 seconds, sometimes less.

**Comment by mani-monaj** [comment]
@eokeeffe @oars have you tried running the driver on another computer?

**Comment by eokeeffe** [comment]
found that reconnecting the WiFi while the application is running seems to do the trick
sometimes the video starts alright and sometimes doesn't .
System setup: Ubuntu 12.04 ROS Groovy&  AR Drone driver is 2.4.8

**Comment by oars** [comment]
@mani-monaj I just tried it on a different machine and it does work. At this point I am unsure of what the cause of the problem can be. I believe it might be the wireless card on one of the machines. Looking at the latency in the connection the ping between the machine and the drone varies drastically and sometimes reaches over 2000 ms. The odd part is that the card performs properly connecting to any other devices.

@eokeeffe I tried reconnecting to the drone network in multiple ways and was able to get the system running (although some minor frame drops still occurred) a couple of times. I was however unable to identify a pattern as I was not able to reliably create a stable connection.
